DNDinventory/SettingsFileHandler.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Xml.Serialization;
+using System.IO;
+using System.Threading.Tasks;
+
+namespace DNDinventory {
+    public class SavedSettings {
+        // Set defaults here
+        public string host = "localhost";
+        public string port = "100";
+        public string outputFolder = "Transfers";
+    }
+
+    public class SettingsFileHandler {
+        public SavedSettings currentSettings = new SavedSettings();
+        XmlSerializer ser = new XmlSerializer(typeof(SavedSettings));
+        
+        public void WriteToXml (string filename) {
+            TextWriter writer = new StreamWriter(filename);
```

Comment:
Use using(){} of C# this way when an error occurs the writer will be closed. The current implementation will force exit the function.